test/auro-combobox.test.js

```diff
@@ -506,7 +507,7 @@ async function noFilterFixture() {
 }
 
 function setInputValue(el, value) {
-  const auroInput = el.shadowRoot.querySelector('auro-input');
+  const auroInput = el.shadowRoot.querySelector('combobox-input');
```

Comment:
we need to preserve `auro` in the component names.

Let's rename all these to `auro-comboboxinput`

Reply:
> we need to preserve `auro` in the component names.
> 
> Let's rename all these to `auro-comboboxinput`

I made these changes as per @blackfalcon's request. We can discuss as a team which naming convention we would like.